FRENCTools_lib/cfrenctools/shared/create_xgrid.h

```diff
@@ -20,10 +20,16 @@
 #ifndef CREATE_XGRID_H_
 #define CREATE_XGRID_H_
 #ifndef MAXXGRID
+#ifdef use_libMPI
```

Comment:
not sure why this is showing up as changed.  It should have been there in the original create_xgrid.h